apps/admin/src/pages/faq/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { Flex } from '@ceos-fe/ui';
+import { QueryClient, dehydrate, useQuery } from '@tanstack/react-query';
+import { CategoryType, ResponseInterface, faqApi } from '@ceos-fe/utils';
+
+interface FaqResponse {
+  categoryFaqList: {
+    id: number;
+    category: CategoryType;
+    question: string;
+    answer: string;
+  };
+}
+
+const Faq = () => {
+  const { data, isLoading, isSuccess } = useQuery<
+    ResponseInterface<FaqResponse>
+  >(['admin', 'faq'], () => faqApi.GET_FAQ('RECRUIT'));
+
+  console.log(data);
+
+  return <Flex>FAQ</Flex>;
+};
+
+export const getStaticProps = async () => {
+  try {
+    const queryClient = new QueryClient();
```

Comment:
app.tsx에서 선언한 queryClient를 사용하지 않고 새로 선언해 주는 이유가 있나여?.?

Reply:
음 각 페이지별로 요청을 분리하기 위해서 새로운 queryClient를 선언한다고 [공식문서](https://tanstack.com/query/latest/docs/react/guides/ssr?from=reactQueryV3&original=https%3A%2F%2Ftanstack.com%2Fquery%2Fv3%2Fdocs%2Fguides%2Fssr)에 되어있길래 따라했는데,,,ㅎ 저 queryClient를 계속 재사용할 수 있나용?

Reply:
움 페이지 간에 데이터가 공유될 필요가 없을 때는 저렇게 사용하면 될 거 같은데,, 모든 서버 상태를 한꺼번에 관리하기에는 app.tsx에서 선언한 queryClient를 다시 사용하는 게 좋을 거 같긴 해여. . . 

```suggestion
    const queryClient = QueryClient.getQueryClient();
```

이런 식으로 하믄 다시 가져올 수 있지 않나여?.?